cmd/deployment.go

```diff
@@ -88,6 +88,22 @@ var (
 	deploymentUserUpdateExample = `
 # Update a workspace user's deployment role
   $ astro deployment user update --deployment-id=xxxxx --role=DEPLOYMENT_ROLE <user-email-address>
+`
+	deploymentTeamAddExample = `
+# Add a workspace team to a deployment with a particular role
+  $ astro deployment team add --deployment-id=xxxxx --team-id=<team-id> --role=DEPLOYMENT_ROLE
+`
+	deploymentTeamRemoveExample = `
+# Remove team access to a deployment
+	$ astro deployment team remove <team-id> --deployment-id=xxxxx
```

Comment:
Akward indent here, hard to read for users and lack of consistency accross our different examples, can you move them all to the begining of the line (no indent) please ?

Reply:
done